src/node/rpc/frontend.h

```diff
@@ -347,7 +347,7 @@ namespace ccf
         {
           func(args);
 
-          if (ctx->response_is_error())
+          if (!ctx->get_apply_writes())
```

Comment:
since  `get_apply_writes` returns a boolean maybe rename to `writes_applied` or `check_writes_applied` something similar

Reply:
This is asking whether we _should_ apply writes, not whether we have applied them in the past. I considered `get_applyable`, but I think `set_apply_writes` is clearer for app-writers and then this is symmetric.

Reply:
then maybe `should_apply_writes`?

Reply:
I prefer `get`/`set`, unless this is significantly less readable?

Reply:
Well get implies that we are `getting` some internal state of the context, where as `should/check` etc implies making a decision based on something. Without knowing the context it looks like you are `getting` what you are about to write

Reply:
Fair, I hadn't twigged this looks like _writes_ as a noun, have changed it to `should_`